instrumentation/opentelemetry-instrumentation-grpc/tests/test_client_interceptor.py

```diff
@@ -269,6 +272,32 @@ def test_error_stream_stream(self):
             trace.StatusCode.ERROR,
         )
 
+    def test_client_interceptor_falsy_response(
+        self,
+    ):  # pylint: disable=no-self-use
+        """ensure that client interceptor closes the span only once even if the response is falsy."""
+
+        with mock.patch.object(SdkSpan, "end") as span_end_mock:
+            tracer_provider, _exporter = self.create_tracer_provider()
+            tracer = tracer_provider.get_tracer(__name__)
+
+            interceptor = OpenTelemetryClientInterceptor(tracer)
+
+            def invoker(_request, _metadata):
+                return {}
+
+            request = Request(client_id=1, request_data="data")
+            interceptor.intercept_unary(
+                request,
+                {},
+                _UnaryClientInfo(
+                    full_method="/GRPCTestServer/SimpleMethod",
+                    timeout=None,
+                ),
+                invoker=invoker,
+            )
+            assert span_end_mock.call_count == 1
```

Comment:
```suggestion
            self.assertEqual(span_end_mock.call_count, 1)
```

And you can drop the no-self-use